test/intl402/Temporal/ZonedDateTime/from/timezone-case-insensitive.js

```diff
@@ -0,0 +1,619 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Ditto.

Reply:
I added my own copyright line for all new files I created here in this PR. There were a lot more than just the few you flagged.